jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -156,13 +152,15 @@ properties:
             title: "Date-time start of data acquisition"
             fits_keyword: DATE-BEG
             blend_table: True
-            blend_rule: mindate
+            blend_rule: mindatetime
           date_end:
             type: string
             title: "[yyyy-mm-dd] UTC date at end of exposure"
             fits_keyword: DATE-END
             blend_table: True
-            blend_rule: maxdate
+            # Despite what the title might lead you to believe, this keyword
+            # actually contains a full datetime string.
```

Comment:
We'll need to fix the keyword title at some point, to keep in sync with what's now in the keyword dictionary (it's been updated there to reflect the fact that it's now a full date-time value), but let's hold off on doing that temporarily, because it'll cause MASSIVE numbers of failures in our regression tests that we don't want to deal with when getting ready for another release.